src/main/java/edu/harvard/iq/dataverse/DataFile.java

```diff
@@ -303,7 +303,18 @@ public Boolean getDeleted() {
     public void setDeleted(Boolean deleted) {
         this.deleted = deleted;
     }
-    
+
+    @Transient
+    private boolean unpublished;
+
+    public boolean getUnpublished() {
```

Comment:
If this is needed (see other comment), can this be a Boolean with the logic to calculate it added here and done only once (if unpublished == null)?